theano/gpuarray/opt.py

```diff
@@ -51,6 +51,7 @@
 
 
 gpu_optimizer = EquilibriumDB()
+gpu_optimizer2 = EquilibriumDB()
```

Comment:
We're using a different instance because it'd make us distinguish between the performance of this optimizer from the others while profiling ? Or is it because we're overusing the existing instance ? (The reason for the max'ed out error).